contracts/interfaces/IGitConsensus.sol

```diff
@@ -163,30 +163,20 @@ interface IGitConsensus is IGitConsensusErrors, IGitConsensusEvents, IGitConsens
         uint256[] calldata values
     ) external returns (bytes20 tagHash);
 
-    /// @notice Get the owner address of a git commit.
-    /// @param commitHash The hash of the git commit. If the commit that generated this hash does
+    /// @param gitHash The hash of the git commit. If the commit that generated this hash does
     ///     not exist, it means that no corresponding commit was ever added to the contract.
-    /// @return ownerAddr The address of the owner of the commit. This is equal to the address
-    ///     that was included in the commit message. If this commit hash was not notarized
-    ///     via `addCommit()`, then the address returned is a zero address.
-    function commitAddr(bytes20 commitHash) external view returns (address ownerAddr);
+    /// This hash can either come from a release hash or a given commit hash.
+    /// @return hashAddr The address that corresponds to the commit or tag. In the case of a commit,
+    /// this is equal to the address that was included in the commit message which is generally used
+    /// to represent the owner of the commit. In the case of a tag, this is equal to the address that
+    /// was included in the tag message, which should be the token address. If the gitHash
+    /// was not notarized via `addCommit()` or `addRelease()`,
+    /// then the address returned is a zero address.
+    function hashAddr(bytes20 gitHash) external view returns (address addr_);
 
-    /// @notice Check if a commit hash exists, indicating that the corresponding commit has been
+    /// @notice Check if a hash exists, indicating that the corresponding commit has been
```

Comment:
/// @notice Check if a hash exists, indicating that the corresponding commit has been
    ///     notarized in the contract via `addCommit()` previously.
    /// @param gitHash The hash of the git commit.
    /// @return exists `true` if the commit hash exists, `false` otherwise.

to 

    /// @notice Check if a git hash exists, indicating that the corresponding commit or tag has been
    ///     notarized in the contract via `addCommit()` or `addRelease()` previously.
    /// @param gitHash The hash of the git commit or tag.
    /// @return exists `true` if the hash exists, `false` otherwise.